pkg/syfs/syfs.go

```diff
@@ -75,6 +77,12 @@ func configDir(dir string) string {
 		} else {
 			homedir = user.HomeDir
 		}
+	} else if homedir != nonexistentDir && !fs.IsReadable(homedir) {
+		homedir = nonexistentDir
```

Comment:
I think that using `/` as a default will be less surprising to users, since that's what happens with ssh.  `/` always has to be readable.

Reply:
okay. sure.